test/ably/rest/restauth_test.py

```
@@ -97,19 +98,19 @@ async def test_request_basic_auth_header(self):
     async def test_request_basic_auth_header_with_client_id(self):
         ably = AblyRest(key_secret='foo', key_name='bar', client_id='client_id')
 
-        with mock.patch.object(AsyncClient, 'send') as get_mock:
+        with mock.patch.object(AsyncSession, 'send') as get_mock:
             try:
                 await ably.http.get('/time', skip_auth=False)
             except Exception:
                 pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)